backend/sirius-web-collaborative-api/src/main/java/org/eclipse/sirius/web/collaborative/api/services/ChangeDescription.java

```diff
@@ -0,0 +1,44 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.collaborative.api.services;
+
+import java.util.Objects;
+import java.util.UUID;
+
+/**
+ * The change description is used to express the changes that have been performed in response to an input in the
+ * collaborative layer. It helps the various representations event processors among others determined if they are
+ * impacted by a change and thus if they should refresh the representation that they are managing.
+ *
+ * @author sbegaudeau
+ */
+public class ChangeDescription {
```

Comment:
A simple `toString` would be useful for debugging I think.

Reply:
Indeed, I'll add that